### DISCUSSION TITLE
Reddit has been made for Friction

### DISCUSSION THREAD

**Discussion by Rybread99** [friction2d/friction#95] [General]
I don’t know if this is allowed but I have created a reddit for all things Friction 2d related here is the link: https://www.reddit.com/r/frictiongraphics/s/FrBJPlFLiJ

**Comment by rodlie** [comment]
I don't use reddit, but created a new user (oarodlie) and joined. Don't know how active I will be (there are too many social sites to follow/use).

*Sorry for the late reply*

**Comment by rodlie** [comment]
I added it to the footer on https://friction.graphics

https://github.com/friction2d/friction2d.github.io/commit/563d26e80d2ffb3752443a1015070138eb57a3ec

Thanks :+1:
